### ISSUE TITLE
Automating certificate management in Edge Browser through programming

### ISSUE THREAD

**Issue by yupeijian9** [MicrosoftEdge/DevTools#150]
Hello developer，  
When it comes to managing certificates in the Edge browser, the "Manage certificates" option can be found in the "Privacy, search, and services" setting of Edge. By opening the certificate manager, various operations related to certificate management can be performed manually,
For example: 
option1: viewing certificate details and certificate chain details; 
option2: importing and deleting certificates;
option3: checking certificate status, 
and so on. 
May I ask how to achieve automation of these operations through programming, including loading certificates, parsing certificates, building certificate chain, verifying certificates, obtaining verification results, and deleting certificates, replacing manual operations (options 1, 2, 3)?
In other words, how can all the available functions in the certificate manager be implemented through programming and run automatically through the program to replace laborious manual operations?
Thank you.

**Comment by captainbrosset** [comment]
Thanks for your question. However, this repository is specific to DevTools. DevTools is a developer feature in Edge which is unrelated with the certificate manager.
Can you please send your feedback again by using the browser's feedback mechanism? Here's how:

1. Click **Settings and more** (the `...` button in the top-right corner of the browser).
2. Click **Help and feedback**.
3. Click **Send feedback**.

You might also be interested in asking on our Edge Insiders community forum instead: https://techcommunity.microsoft.com/t5/microsoft-edge-insider/ct-p/MicrosoftEdgeInsider

Thank you for your understanding.
